### ISSUE TITLE
Document emoji add endpoint

### ISSUE THREAD

**Issue by G-Rath** [slackhq/slack-api-docs#94]
According to the comments of #28, `/api/emoji.add` is an endpoint that can be used for adding emojis.

However, this endpoint isn't documented anywhere!

**Comment by ErikKalkoken** [comment]
This is an internal endpoint that only works with Slack internal tokens (xoxs...), but not with any normal token (user, bot. legacy). You can fetch internal tokens from the browser, but that's a hack and not recommended. So since it can not really be used by any Slack app there is not much point in documenting it, unless Slack opened this endpoint up for normal tokens.

**Comment by jonmcoe** [comment]
Agreed. No sense in this (at least for me) until an approach with different auth is added, as suggested in #95

**Comment by davidbitton** [comment]
It would be nice to be able to bulk load emojis, That's how I plan on using it.

**Comment by jonmcoe** [comment]
@davidbitton something like the project below can help you for isolated bulk uploads. 

https://github.com/smashwilson/slack-emojinator/blob/master/upload.py

It works well. The downside is you have to take short lived personal authentication (xoxs) from you browser session. So you couldn't really make a chatbot, a web app that does image resizing for joy x etc

**Comment by davidbitton** [comment]
@jonmcoe i'll check it out. thanks
